### ISSUE TITLE
Accessibility focus trap on Travel Advice screen

### ISSUE THREAD

**Issue by hspinks** [WorldHealthOrganization/app#941]
Accessibility Audit via @avkvirtru 

**Problem:** Focus trap. Once I navigate to this red banner, I cannot navigate to the header / back button nav area via VoiceOver. I can only navigate back to the content.

**Acceptance Criteria:** Should be able to navigate back to the header / back button area.

---
- [x] Searched the [existing issues](https://github.com/WorldHealthOrganization/app/issues) to ensure you are not creating a duplicate.
- [x] Followed the [Contributor Guidelines](https://github.com/WorldHealthOrganization/app/blob/master/docs/CONTRIBUTING.md).
